src/pages/index.tsx

```diff
@@ -1,20 +1,21 @@
 import type { NextPage } from "next";
 import styled from "styled-components";
-import { useDispatch, useSelector } from "react-redux";
+import { useAppDispatch, useAppSelector } from "../store/hooks";
 import {
   decrement,
   increment,
   clear,
   incrementByAmount,
   changeCounter,
+  getTestData,
 } from "../store/reducer/counterSlice";
 import HeaderLayout from "../layouts/HeaderLayout";
 import { Button, Box, Flex, Heading, Text } from "@chakra-ui/react";
 import ChangeCourtSize from "@/components/ChangeCourtSize";
 
 const Home: NextPage = () => {
-  const count = useSelector(changeCounter);
-  const dispatch = useDispatch();
+  const count = useAppSelector(changeCounter);
```

Comment:
It seems like this redux selector is for getting value from the store so the name of the selector might need to change.